Libraries/Text/TextInput/Multiline/RCTUITextView.m

```diff
@@ -56,7 +56,14 @@ - (instancetype)initWithFrame:(CGRect)frame
     self.backgroundColor = [RCTUIColor clearColor]; // TODO(macOS ISS#2323203)
     self.textColor = [RCTUIColor blackColor]; // TODO(macOS ISS#2323203)
     // This line actually removes 5pt (default value) left and right padding in UITextView.
+#if !TARGET_OS_OSX // TODO(macOS ISS#2323203)
     self.textContainer.lineFragmentPadding = 0;
+#else
+    // macOS has a bug where setting this to 0 will cause the scroll view to scroll to top when
+    // inserting a newline at the bottom of a NSTextView when it has more rows than can be displayed
+    // on screen.
```

Comment:
Is there a radar filed for this and/or another existing public write-up on the bug? If so, can you leave a comment here linking to it?

Reply:
I didn't file a Radar. I'll add it my to todo list, but it's not a high priority now.